docs/docs/configuration/theming.mdx

```diff
@@ -62,12 +62,13 @@ THEME_DARK = {
     }
 }
 
-# Theme behavior settings
-THEME_SETTINGS = {
-    "enforced": False,          # If True, forces default theme always
-    "allowSwitching": True,     # Allow users to switch between themes
-    "allowOSPreference": True,  # Auto-detect system theme preference
-}
+# Enable UI-based theme administration for admins
+ENABLE_UI_THEME_ADMINISTRATION = True
+
+# To force a single theme on all users, set THEME_DARK = None
+# When both THEME_DEFAULT and THEME_DARK are defined:
+# - Users can manually switch between themes
+# - OS preference detection is automatically enabled
```

Comment:
I think here we are losing on flexibility. We know of use cases in Embedded where users want to be able to control what theme mode is applied. They might have a specific default and dark theme but also the need of setting it programmatically rather than relying on machine preference. Would that ability be lost with this change?

Reply:
Thinking about the embedder's perspective, I believe they can set whatever theme they'd like programmatically, the question is whether that's sufficient or whether we want for them to be able to reference arbitrary themes from the CRUD, including the system ones and the auto-switch. For now, I'm thinking the simplest is clearly to have them manage which theme they want to set programmatically, and in isolation from the CRUD: basically if you want to control the theme you pass in a theme object as you embed. 

I guess the embedded sdk could also have a layer to retrieve theme definitions from the Theme CRUD using the API too, like fetch whatever theme and use it...

Reply:
A simple thing to imagine is that they would have their own mode picker in the host app (light/dark/use-os), in which case they hold the state and have to manage their own logic as to how to couple the embedded dashboard's theme fetching, and can't rely on our logic / picker to do the same thing. Seems that logic of deciding which Superset-hosted theme to use belongs on their side and is best managed as code as opposed to configuration.